app/modules/conversations/conversation/conversation_controller.py

```
@@ -96,11 +96,11 @@ async def post_message(
             raise HTTPException(status_code=500, detail=str(e))
 
     async def regenerate_last_message(
-        self, conversation_id: str, node_ids: List[NodeContext] = []
+        self, conversation_id: str, node_ids: List[NodeContext] = [], stream: bool = True
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)